src/main/java/domain/InvoiceDetails.java

```diff
@@ -44,15 +44,15 @@
     private List<Long> locationPointsIds;
 
     private String description;
-    private BigDecimal price;
+    private int price; // Price in cents
     private long distance;
 
     public InvoiceDetails() { }
 
-    public InvoiceDetails(ArrayList<TranslocationDto> locationPoints, String description, BigDecimal price) throws InvoiceException {
+    public InvoiceDetails(ArrayList<TranslocationDto> locationPoints, String description, int categoryPrice) throws InvoiceException {
         if(locationPoints == null || locationPoints.size() < 2) { throw new InvoiceException("No locationPoints provided for InvoiceDetails or only 1 provided."); }
         if(description.isEmpty()) { throw new InvoiceException("Please provide a description for this InvoiceDetail."); }
```

Comment:
Zag atm dat description hardcoded "complete journey" is. Opzich prima maar mss kunnen we dit weglaten of vervangen door bijvoorbeeld een automatisch genenereerde text met voortuiggegevens en datums? Niet perse fout maar mss iets voor later.